.all-contributorsrc

```diff
@@ -2607,6 +2607,18 @@
         "code"
       ]
     },
+    {
+      "login": "ShreeshaM07",
+      "name": "Shreesha M",
+      "avatar_url": "https://avatars.githubusercontent.com/u/120820143?s=400&v=4",
+      "profile": "https://github.com/ShreeshaM07",
+      "contributions": [
+        "bug",
+        "code",
+        "test",
+        "doc"
```

Comment:
I would remove this, together with the change to the doctoring. See the comment, I added regarding the docstring

Reply:
Should I remove all the four bug code test and doc?

Reply:
Sorry for the unclearities. I would suggest to remove docs here to avoid confusion. 

But you can create a new PR that just adds your doc badge. In this new PR, please refer to the PR where you added the docs.

Reply:
okay I have removed the `doc` in `.all-contributorsrc`. Also I think there is some issue with test-mlflow and test-pandas1 thats been raised it is not working for me either.